tests/interu.yaml

```diff
@@ -0,0 +1,24 @@
+runners:
```

Comment:
Todo in a new templating run

```suggestion
---
runners:
```

Reply:
Good point, but this file will is not and will not be templated.

Reply:
Na, as I mentioned, it can go into the next templating run.